drivers/gpu/drm/amd/display/dc/core/dc_stream.c

```diff
@@ -505,6 +505,10 @@ bool dc_stream_remove_writeback(struct dc *dc,
 		dm_error("DC: writeback pipe is invalid!\n");
 		return false;
 	}
+    if (stream->num_wb_info > MAX_DWB_PIPES) {
```

Comment:
We haven't touched this file in this repo to my knowledge, so I suspect the correct fix is a fresh pull from upstream.

Reply:
I removed this commit (rebase)